style.css

```diff
@@ -0,0 +1,974 @@
+body,
+h1,
+h2,
+h3,
+h4,
+p { margin: 0; }
+
+body {
+	min-height: 100vh;
+	line-height: 1.5;
+}
+
+img {
+	max-width: 100%;
+	display: block;
+}
+
+a { text-decoration: none; }
+
+/* Desktop */
+
+	/* Шапка */
+		.header,
+		.header__menu,
+		.header__wrapper_item {
+			display: flex;
+			justify-content: space-between;
+		}
+
+		.header {
+			align-items: center;
+			height: 104px;
+			margin: 0 auto;
+			margin-bottom: 30px;
+		}
+
+		.header__wrapper_logo {
+			display: flex;
+			justify-content: center;
+			flex: .21;
+		}
+
+		.header__menu { flex: .668; }
+
+		.header__wrapper_item { flex: .486; }
+
+		.header__link,
+		.button__item_link { color: #000; }
+
+		.header__wrapper {
+			display: flex;
+			justify-content: flex-start;
+			flex: .294;
+		}
+
+		.header__wrapper_link { margin-right: 30px; }
+		
+		.burger:hover {	cursor: pointer; }
+		
+		.burger__icon {
+			display: block;
+			width: 25px;
+			height: 2px;
+			margin-bottom: 5px;
+			background-color: #E51937;
+			border-radius: 0 0 5px 5px;
+		}
+		
+		.header__link:hover {
+			border-bottom: 2px solid #E51937;
+			border-radius: 2.5px;
+			color: #E51937;
+		}
+
+	/* Первый экран */
+		.section-main {
+			margin-bottom: 34px;
+			background: url(./Project/tesla-wordmark-red_big.svg) no-repeat;
+			background-position: top;
+			background-size: 95%;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Project_page/pullRequest?prid=5536520)